consensus/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "aleph-bft"
-version = "0.20.0"
+version = "0.21.0"
```

Comment:
I think you haven't technically changed any interfaces, so this should be just a patch version bump. Same for all(?) other bumps.